dev/ci/README-developers.md

```diff
@@ -43,9 +43,10 @@ See also [`test-suite/README.md`](../../test-suite/README.md) for information ab
 ### Breaking changes
 
 When your PR breaks an external project we test in our CI, you must
-prepare a patch (or ask someone to prepare a patch) to fix the
-project. There is experimental support for an improved workflow, see
-[the next section](#experimental-automatic-overlay-creation-and-building), below
+prepare a patch (or ask someone —possibly the project author— to
```

Comment:
```suggestion
prepare a patch (or ask someone—possibly the project author—to
```
I think generally there are either spaces on both sides, or spaces on neither side; I've never seen dashes like this with spaces only on one side (I've only seen that when combining the suffix of two hypenated words, such as in "brother- and sister-in-law")

Reply:
Note that these are em-dashes, so not the same character as in "brother-in-law". You might still be right though.

Reply:
Yes, not the same character.  But see, e.g., https://www.thepunctuationguide.com/em-dash.html
> Spaces with the em dash
> The em dash is typically used without spaces on either side, and that is the style used in this guide. Most newspapers, however, set the em dash off with a single space on each side. Example Most newspapers — and all that follow AP style — insert a space before and after the em dash.